ncurses.go

```diff
@@ -253,6 +253,11 @@ func ResizeTerm(nlines, ncols int) error {
 	return nil
 }
 
+// Set the ESCDELAY
+func SetEscDelay(size int) {
```

Comment:
I don't have a machine available for testing right now but it looks like this function is an ncurses only extension and is not available in PDCurses. In order to accept this patch, I would need a NOOP wrapper. For examples see: https://github.com/rthornton128/goncurses/blob/master/goncurses.c

Don't forget to add a header to goncurses.h and use `goncurses_set_escdelay()` here.